source/genericobject/index.rst

```diff
@@ -49,8 +49,18 @@ Install the Plugin
 
 * Uncompress the archive.
 * Move the ``genericobject`` directory to the ``<GLPI_ROOT>/plugins`` directory
-* Navigate to the *Configuration > Plugins* page,
-* Install and activate the plugin.
+* Navigate to the *Configuration > Plugins* page
+* Install and activate the plugin
+* Set rights on plugin directories :
+    
+.. code-block:: bash
+   
+   chown -R myaccount:apache <GLPI_ROOT>/plugins/genericobject
```

Comment:
This should not be required. Plugins must not write in their own directory; since GLPI provides `files/_plugins` for that.

Reply:
You're right but I will change line 59 in place of 58.

Reply:
why does this need to be changed?